tests/unit/backend/service/test_MetadataLookupService.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+import pytest
+from snapred.backend.service.MetadataLookupService import MetadataLookupService
+
+
+class TestMetadataLookupService(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.instance = MetadataLookupService()
+        super().setUpClass()
+
+    def setUp(self):
+        return super().setUp()
+
+    def test_verifyMultipleRuns(self):
```

Comment:
You need a test to cover the scenario that `IntArrayProperty` can parse the string to a list of run numbers, but the list contains one valid and one invalid run number, and make sure only the valid run is returned.